### ISSUE TITLE
Organization Admin APIs

### ISSUE THREAD

**Issue by alishaz-polymath** [calcom/docs#113]
Add all details regarding org admin APIs and access control

**Comment by PeerRich** [comment]
yeah this needs a dedicated page for admin API, its too important. also ask @shirazdole about what the best way is to request access (I imagine link to https://cal.com/sales )
